packages/app/src/app/pages/Dashboard/Components/Selection/index.tsx

```diff
@@ -640,7 +640,6 @@ export const SelectionProvider: React.FC<SelectionProviderProps> = ({
         id="selection-container"
         onContextMenu={onContainerContextMenu}
         css={css({
-          paddingTop: page === 'recent' ? 0 : 8, // In the recent page, this component is nested so the padding top isn't needed.
```

Comment:
This padding is needed otherwise the boundaries of the item aren't clickable:

![Kapture 2023-02-01 at 22 27 30](https://user-images.githubusercontent.com/24959348/216208556-2d4ae7bd-4627-4792-8796-e17b4839b850.gif)